packages/core/src/app/payment/storedInstrument/isInstrumentFeatureAvailable.ts

```diff
@@ -5,20 +5,23 @@ export interface IsInstrumentFeatureAvailableState {
     customer: Customer;
     isUsingMultiShipping: boolean;
     paymentMethod: PaymentMethod;
+    shouldSavingCardsBeEnabled?: boolean;
 }
 
 export default function isInstrumentFeatureAvailable({
     config,
     customer,
     isUsingMultiShipping,
     paymentMethod,
+    shouldSavingCardsBeEnabled = true,
```

Comment:
Maybe better to change this value to `undefined` by default and make it like `shouldSavingCardsBeDisabled`  or `disabledSavingCards` ? what do you think?

Reply:
As we discussed on private message, I personally like keeping `shouldSavingCardsBeEnabled` set to `true` as default as it indicates that checkbox should be visible by default. But it can be also set as `undefined` if there is such a need